internal/api_admin/server.go

```diff
@@ -323,6 +328,81 @@ func (s *Server) RevokeConnectionCredentials(ctx context.Context, request Revoke
 	return RevokeConnectionCredentials202JSONResponse{Message: "Credentials revocation request sent"}, nil
 }
 
+// CreateLink - creates a link for issuing a credential
+func (s *Server) CreateLink(ctx context.Context, request CreateLinkRequestObject) (CreateLinkResponseObject, error) {
+	var expiresAt *time.Time
+	if request.Body.ExpirationDate != nil {
+		layout := "2006/01/02"
+		t, err := time.Parse(layout, *request.Body.ExpirationDate)
```

Comment:
It is better to use date format modifier in the yaml so we don't need to parse it here and use standard dates.